audit-service/src/main/java/uk/gov/gchq/palisade/service/audit/model/AuditMessage.java

```diff
@@ -39,17 +39,30 @@
 @JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY)
 public class AuditMessage {
 
-    protected static final ObjectMapper MAPPER = new ObjectMapper();
-
-    protected final String userId; //Unique identifier for the user.
-    protected final String resourceId;  //Resource that that is being asked to access.
-    protected final JsonNode context;   //Relevant context information about the request.
-    protected final String serviceName;  //service that sent the message
-    protected final String timestamp;  //when the message was created
-    protected final String serverIP;  //the server IP address for the service
-    protected final String serverHostname;  //the hostname of the server hosting the service
-    protected final JsonNode attributes;  //JsonNode holding Map<String, Object> holding optional extra information
-
+    @SuppressWarnings("java:S2039")
+    static final ObjectMapper MAPPER = new ObjectMapper();
```

Comment:
what are we supressing here?

Reply:
can you add a small javadoc with the rule, // supress member variable visibility rule, or something similar